messages/ownship/steering_unit.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
I think that the contents of this file should be included in [rudder observation message](https://github.com/MO-RISE/brefv/blob/main/messages/observations/rudder.json). According to me, an observation should contain everything that is needed to interpret (and draw in the frontend) the sensor and its information.